### DISCUSSION TITLE
Message "Hi there! How can I help?, where can I modify it?

### DISCUSSION THREAD

**Discussion by claclin** [FlowiseAI/Flowise#1654] [Q&A] [answered]
I would like to modify the wellcome message appears every time I open the chat. How can I do that? Thanks in advance.

**Comment by Jaredude** [comment, marked as answer]
You'd modify the welcome message on the bot init. [You can reference full docs here](https://github.com/FlowiseAI/FlowiseChatEmbed#configuration).

**Comment by claclin** [comment]
Thanks a lot.

El jue, 1 feb 2024 a las 15:49, Jared Tracy ***@***.***>)
escribió:

> You'd modify the welcome message on the bot init. You can reference full
> docs here <https://github.com/FlowiseAI/FlowiseChatEmbed#configuration>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/FlowiseAI/Flowise/discussions/1654#discussioncomment-8335820>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2LHOFFR2AK62KW4NXPFPE3YROTRJAVCNFSM6AAAAABCUNWP6CVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGMZVHAZDA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Saludos,

Claudio
